AzureCalling/Views/GroupIdShareItem.swift

```diff
@@ -6,27 +6,26 @@
 import UIKit
 import LinkPresentation
 
-class JoinIdShareItem: NSObject, UIActivityItemSource {
+class GroupIdShareItem: NSObject, UIActivityItemSource {
+    private static let shareTitle: String = "Share Group Call ID"
 
-    private var joinId: String
-    private var shareTitle: String
+    private var groupId: String
 
-    init(joinId: String, shareTitle: String) {
-        self.joinId = joinId
-        self.shareTitle = shareTitle
+    init(groupId: String) {
+        self.groupId = groupId
     }
 
     func activityViewControllerPlaceholderItem(_ activityViewController: UIActivityViewController) -> Any {
         return ""
     }
 
     func activityViewController(_ activityViewController: UIActivityViewController, itemForActivityType activityType: UIActivity.ActivityType?) -> Any? {
-        return joinId
+        return groupId
     }
 
     func activityViewControllerLinkMetadata(_: UIActivityViewController) -> LPLinkMetadata? {
         let metadata = LPLinkMetadata()
-        metadata.title = shareTitle
+        metadata.title = GroupIdShareItem.shareTitle
         return metadata
     }
 }
```

Comment:
We can keep the changes in `JoinIdShareItem`. There will be less conflicts when we merge later on.